### ISSUE TITLE
TypeError at KeywordCall with variable, but without keyword

### ISSUE THREAD

**Issue by d-biehl** [MarketSquare/robotframework-robocop#318]
if you have a .robot file with this content:

```robotframework
*** Test Case ***
first
    ${a}    ${b}    # no Keyword here!!!

*** Keywords ***
get something
    [Return]    1    2
```

and run robocop with that file, an TypeError is raised:

      File "c:\develop\robot\robotframework-robocop\robocop\checkers\errors.py", line 64, in visit_KeywordCall
      match = self.setting_pattern.match(node.keyword)
    
    TypeError: expected string or bytes-like object.

If I fix this, another error occurs:

      File "c:\develop\robot\robotframework-robocop\robocop\utils\utils.py", line 50, in normalize_robot_name
        return name.replace(' ', '').replace('_', '').lower()
    AttributeError: 'NoneType' object has no attribute 'replace'

**Comment by bhirsz** [comment]
Thanks for reporting - we cover this case in some rules but like you noticed it was missed in those two rules. Your PR looks good although there is no test data added - I can do it though if you want

**Comment by d-biehl** [comment]
Thanks!!!

And also thanks for the tests ;-)
